### DISCUSSION TITLE
Version 2.3.0

### DISCUSSION THREAD

**Discussion by notheotherben** [SierraSoftworks/tailscale-udm#36] [Announcements]
This release is a big one: it adds experimental support for UniFi OS 2.x and UniFi OS 3.x, which are found on the UDR and UDM-SE.

Installation steps are essentially identical to those for UniFi OS 1.x, however the management script will be found in `/data/tailscale/manage.sh` instead of `/mnt/data/tailscale/manage.sh` on these UniFi 2.x+ devices.

If you run into any problems during installation and/or usage with this version, please don't hesitate to create an issue including the logs of what happened.

## What's Changed
* Added tailscale_update to remove on-boot error by @mhriemers in https://github.com/SierraSoftworks/tailscale-udm/pull/35
* feat: Add initial logic for UDM-SE support by @notheotherben and @DennisGaida in https://github.com/SierraSoftworks/tailscale-udm/pull/33

## New Contributors
* @mhriemers made their first contribution in https://github.com/SierraSoftworks/tailscale-udm/pull/35

**Full Changelog**: https://github.com/SierraSoftworks/tailscale-udm/compare/v2.2.1...v2.3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/SierraSoftworks/tailscale-udm/releases/tag/v2.3.0'>Version 2.3.0</a>.</em>

**Comment by rtrainer** [comment]
Does this work on the UDM Pro now that they are rolling out UniFi OS v2.4 to them?

**Reply by rtrainer** [reply]
I did the install and there was a single error. at the end of the install script, it spit out "context canceled" at if displayed the authentication URL. Everything appears to work despite that message.

**Reply by notheotherben** [reply]
Hi @rtrainer, this should work on the UDM, UDM Pro, UDR, and UDM SE - whether they are running UniFi OS 1.x (as the UDM and UDM Pro did until recently) or UniFi OS 2.x/3.x as the devices do now.

I'm currently running this on my UDM @ 2.4.27 without any issues.

The `context cancelled` error is due to us setting a timeout of 5 seconds for running `tailscale up` and shouldn't cause any issues, but I can certainly look at increasing that timeout to minimize the risk that you see the error.

**Reply by notheotherben** [reply]
Just a note that you should use the latest release version (v2.3.3 at the time of writing) for the best chance of a trouble-free installation.
